docs/react-v9/contributing/command-cheat-sheet.md

```diff
@@ -1,4 +1,82 @@
-This is a list of helpful commands.
+## Executing tasks:
+
+To run tasks(targets) we use `nx run` command. Learn more https://nx.dev/nx-api/nx/documents/run
```

Comment:
I see some commands still use `lage`. Will this change in the future or is there a specific reason to keep `lage` in certain cases?

Reply:
yes - see linked Epic for more context  https://github.com/microsoft/fluentui/issues/30267

and https://github.com/microsoft/fluentui/pull/31949